src/mixins/nft.js

```diff
@@ -440,6 +440,9 @@ export default {
         1
       );
       try {
+        if (this.walletLIKEBalanceFetch) {
+          await this.walletLIKEBalanceFetch;
```

Comment:
Maybe `walletLIKEBalanceFetching` or `walletLIKEBalanceFetchPromise` ? very confusing now since `walletFetchLIKEBalance` and `walletLIKEBalanceFetch` both exists